server/sessions.py

```
@@ -152,6 +175,20 @@ def open(
         session_id = "ses_" + secrets.token_hex(4)
         state, started = ("open", None) if kind == MULTICAST else ("running", now_iso())
         try:
+            if replaces is not None:
+                # ‏BEGIN IMMEDIATE תופס את נעילת הכתיבה **לפני** הקריאה־
+                # כתיבה, ולכן הסגירה וה-INSERT הם פעולה אחת מבחינת כל
```

Ruff (0.16.2)

[error] Contains control characters that can permit obfuscated code

(PLE2502)